src/Audit/Audit.php

```diff
@@ -459,4 +459,38 @@ public function cleanup(string $datetime): bool
 
         return true;
     }
+
+    /**
+     * Add multiple event logs in batch.
+     *
+     * @param array<array{userId: string, event: string, resource: string, userAgent: string, ip: string, location: string, timestamp: string, data?: array<string,mixed>}> $events
+     * @return bool
+     *
+     * @throws AuthorizationException
+     * @throws StructureException
+     * @throws \Exception
+     * @throws \Throwable
+     */
+    public function logByBatch(array $events): bool
```

Comment:
Can we move the definition to immediately after the `log` function, and rename to just `logBatch`

Reply:
Addressed 👌